.vscode/launch.json

```
@@ -19,16 +19,17 @@
       "request": "launch",
       "name": "Jest Current File",
       "program": "${workspaceFolder}/node_modules/.bin/jest",
-      "args": [
-        "--runInBand",
-        "--collectCoverage=false",
-        "${fileBasenameNoExtension}"
-      ],
+      "args": ["--runInBand", "--collectCoverage=false", "${relativeFile}"],
```

Found a better solution: [`--runTestsByPath`](https://jestjs.io/docs/en/cli#runtestsbypath)
```suggestion
      "args": [
        "--runInBand",
        "--collectCoverage=false",
        "--runTestsByPath",
        "${relativeFile}"
      ],
```